app/views/admin/documentations/show.html.haml

```diff
@@ -1 +1,12 @@
-= @documentation.name
+= link_to admin_documentations_path do
+  %span.glyphicon.glyphicon-circle-arrow-left.lead
+.title
+  = @documentation.name
+%table.table.table-striped
+  %thead
+    %tr.info
+      %th abbréviation
+      %th Accès
```

Comment:
Use rather "Couverture" here. ;)
Later, we would like to display an image preview.